packages/react-native-editor/CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ For each user feature we should also add a importance categorization label  to i
 -->
 
 ## Unreleased
-
+[*] [Embed block] Included Link in Block Settings [#36099]
```

Comment:
I think we're missing a new line here to separate the `Unreleased` from the `1.65.0`.